### ISSUE TITLE
Error: java.lang.NoSuchMethodError: java.nio.ByteBuffer.flip()Ljava/nio/ByteBuffer;

### ISSUE THREAD

**Issue by tuzzmaniandevil** [aws/aws-sigv4-auth-cassandra-java-driver-plugin#5]
I'm getting the following error when trying to use this package with OpenJDK 8

```java
Caused by: java.lang.NoSuchMethodError: java.nio.ByteBuffer.flip()Ljava/nio/ByteBuffer;
	at software.aws.mcs.auth.SigV4AuthProvider.<clinit>(SigV4AuthProvider.java:72)
```

**Comment by tuzzmaniandevil** [comment]
This seems to be a common issue: https://github.com/snazy/ohc/issues/70
Seems to be caused when the jar is compiled with JDK9+.
I cloned the repo and compiled it with JDK8 and the issue went away.

**Comment by dchenbec** [comment]
Thank you for the report, we are investigating

**Comment by dchenbec** [comment]
Unfortunately we cannot deploy a fixed version of 4.0.0 because Maven considers releases immutable and remote repositories may not pick up the change. Instead, please use the 4.0.1 version of the plugin, which we have confirmed is built with JDK8.

**Comment by tuzzmaniandevil** [comment]
@dchenbec Thanks for fixing it so quickly. A new version is ok :-)

**Comment by dchenbec** [comment]
For anyone finding this in the future, we found another bug related to JDK8 -> JDK11 breaking API changes that has been fixed in 4.0.2
